docs/organizations/using-coding-standards.md

```diff
@@ -107,6 +103,9 @@ To delete a coding standard:
 
 1.  Click the trash can icon on the coding standard card and confirm.
 
+    !!! note
+        If you delete a coding standard, Codacy restores the previous code pattern configurations for any repositories following the coding standard. However, the repositories keep the toggle status of the tools from the coding standard.
```

Comment:
See [comment above](https://github.com/codacy/docs/pull/1739/files#r1206757043).

Reply:
I'm editing this note like this:

```suggestion
    !!! note
        If you delete a coding standard Codacy restores the previous code pattern configurations for any repositories following the coding standard, but keeps the tool activation status defined by the coding standard.
```

Reply:
Looking good!